src/virtualize.js

```diff
@@ -111,7 +111,7 @@ export default function virtualize(MyComponent) {
       }
 
       if (onChangeIndex) {
-        onChangeIndex(index);
+        onChangeIndex(index, indexLatest);
```

Comment:
I'm not sure it's what user expect. This `indexLatest` is in the `<SwipeableViews />` referential, not in the HOC one.